docs/send-data/kubernetes/troubleshoot-collection.md

```diff
@@ -395,6 +490,268 @@ kube-prometheus-stack:
 
 where `metadata.name` is the value from Argo Application manifest.
 
+### Check metrics content
+
+You can print metrics on stdout of metrics collector and metrics metadata, and validate if they are correct. It may happen that metrics are ingested, but with different metadata than you expect.
+
+In order to print them on stdout, two steps are required:
+
+1. Disable ingesting logs from metrics related pods. This is required to prevent logs ingest spike. Add the following configuration to `user-values.yaml`:
+
+   ```yaml
+   debug:
+     metrics:
+       metadata:
+         stopLogsIngestion: true
+       collector:
+         stopLogsIngestion: true
+   ```
+
+   Then update your collection and wait for all logs collector pods to be redeployed.
+
+2. Enable printing metrics on stdout for metrics related pods, by adding the following to `user-values.yaml`:
+
+   ```yaml
+   debug:
+     metrics:
+       metadata:
+         print: true
+         stopLogsIngestion: true
+       collector:
+         print: true
+         stopLogsIngestion: true
+   ```
+
+3. To revert changes, please perform first step as it is, and after configuration has been propagated to all pods, you can remove all configuration added in this section from the `user-values.yaml`.
```

Comment:
```suggestion
1. Disable ingesting logs from metrics related pods. This is required to prevent logs ingest spike.
   - Add the following configuration to `user-values.yaml`:
     ```yaml
     debug:
       metrics:
         metadata:
           stopLogsIngestion: true
         collector:
           stopLogsIngestion: true
     ```
   - Then update your collection and wait for all logs collector pods to be redeployed.
2. Enable printing metrics on stdout for metrics related pods by adding the following to `user-values.yaml`:
   ```yaml
   debug:
     metrics:
       metadata:
         print: true
         stopLogsIngestion: true
       collector:
         print: true
         stopLogsIngestion: true
   ```
3. To revert your changes, perform first step as-is, then after configuration has been propagated to all pods, you can remove all configuration added in this section from the `user-values.yaml`.
```